### PR TITLE
Add PHP 8 support and drop PHP 7.0 and lower

### PR DESCRIPTION
**Pull request by ben221199**
This Pull Requests adds PHP 8 support with working tests and drops support for PHP 7.0 and lower.

### CODE REVIEW

**Comment by red-led** [comment]
Thanks! Tested it on my code, works good.
Looking forward for a new release with this changes.

**Comment by ben221199** [comment]
@oohnoitz Can you merge it?

**Comment by nikonor909** [comment]
Does the multi-select query with the show meta helper work for you on this thread?

$result = (new SphinxQL($this->conn))
    ->select()
    ->from('rt')
    ->where('gid', 9003)
    ->enqueue((new Helper($this->conn))->showMeta())

For some reason, I don't have meta information as a result. I am using sphinx 3.3.1, php8, laravel8.

**Comment by red-led** [comment]
@nikonor909 it works in `ben221199:php8` branch, but does not in `FoolCode:php8`.

If you want to try version from this PR add this, to your `composer.json`:
```json
  "repositories": [
    {
      "type": "vcs",
      "url": "https://github.com/ben221199/sphinxql-query-builder"
    }
  ],
  "require": {
...
    "foolz/sphinxql-query-builder": "dev-php8",
...
  }
```

Not recommended for production, obviously.

**Comment by nikonor909** [comment]
@red-led thanks.

**Comment by oohnoitz** [comment]
Looks good to me. We'll want to update the CHANGELOG @ben221199 and mark this as 3.0.0 release. Once that is done, I'll merge and tag the release.

**Comment by ben221199** [comment]
Ok, I will do this now.

**Review by oohnoitz — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,48 +1,47 @@
-#### 2.1.0
+#### 3.0.0
+* Added support for PHP 8
```
Comment (red-led) [inline CHANGELOG.md:2]: I think "drop php 7.0 and lower support" part is also worth mentioning.

Comment (ben221199) [reply CHANGELOG.md:2]: I mentioned in the README that you need PHP 7.1 or up.

Comment (ben221199) [reply CHANGELOG.md:2]: But the maintainers are free to add this.

**Comment by red-led** [comment]
@oohnoitz, can we expect 3.0 release on packagist?

**Pull request merged**